src/targets/maven.ts

```diff
@@ -0,0 +1,250 @@
+import { TargetConfig } from '../schemas/project_config';
+import { BaseArtifactProvider } from '../artifact_providers/base';
+import { BaseTarget } from './base';
+import { ConfigurationError } from '../utils/errors';
+import { homedir } from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { exec } from 'child_process';
```

Comment:
Use `spawnProcess` from `system/utils`
```suggestion
import { spawnProcess } from '../utils/system';
```